python/lsst/ts/idl/enums/ESS.py

```diff
@@ -25,12 +25,20 @@
 
 
 class ErrorCode(IntEnum):
-    """Error codes that can be used by the ESS CSC."""
+    """Error codes that can be used by the ESS CSC.
 
-    NotConnected = 1
-    AlreadyConnected = 2
-    NotConfigured = 3
-    BadConfiguration = 4
-    ReadLoopFailed = 5
-    TelemetryError = 6
-    HardwareError = 7
+    Values:
+
+    * InvalidModelConfig: the configuration for one or more models
```

Comment:
I don't see this value in the list.

Reply:
Good catch. It turns out it wasn't useful -- invalid data client config is caught at the same time as any other invalid config and handled by the the parent class (ConfigurableCsc).